test/e2e/scaffold/k8s.go

```diff
@@ -554,3 +555,70 @@ func (s *Scaffold) GetKubernetesClient() *kubernetes.Clientset {
 	assert.Nil(ginkgo.GinkgoT(), err, "get kubernetes client")
 	return client
 }
+
+func (s *Scaffold) CreateApisixRouteByChever(routeID string, body []byte) error {
```

Comment:
what does `ByChever` mean?

Reply:
Just for demo, I found there has another contributor has write this same function. 
So I  want to use his code after his PR was merged.